azure-pipelines.yml

```
@@ -247,6 +228,30 @@ jobs:
     condition: eq(variables['Build.Reason'], 'PullRequest')
     displayName: Check Licence Headers
 
+- job: slow_lints
+  displayName: Perform Slower Lints
```

How about aligning the two lint jobs in name, e.g. rename the existing one "Run code quality checks (quick lint)", and this one:

```suggestion
  displayName: Run code quality checks (in-depth lint)
```